### DISCUSSION TITLE
Server Error When Connecting to Netlify

### DISCUSSION THREAD

**Discussion by tdarais** [GetPublii/Publii#1575] [Help]
Hi,
I've just moved from Windows 10 to EndeavourOS LXQT desktop.  I was able to bring over to my Linux system copies of my sites (as well as backups of my sites). 

I've run into issues connecting back up with Netlify.  So far as I know I have a good Site ID and Netlify token.  When testing the connection I get the following message:  _Error! Application was able to connect with your server but was unable to store files.  Please check file permissions on your server:  Cannot read properties of undefined (reading 'body')._

Any ideas?  I don't know where to begin.  I couldn't find anything in error on Netlify.  I'm wondering if the app has the proper permissions.  I have discovered that if I use the right Arch install package the app installs to the location stated in the Publii Documentation for installation, and I have used that Arch package.  

Per the installation documentation:  _The Linux version of Publii stores the app files that provide the app functionality and interface in the /opt/Publii/resources/app/ folder. Users that wish to make changes to the core functionality of the app may make their changes in these files, though care should be taken; making such changes can result in instabilities or crashes._

Any help will be much appreciated.  I'm loving the move to Linux and this is one of the last things I need to do to finish out my installation.

Tom D.

**Comment by tdarais** [comment]
Hi, 
I discovered the reason for the above described error.  I had a leading space in front of the Site ID when copying it over to the Publii server settings field.  It would have been helpful is the error said something like "Invalid Site ID or something to that effect."   

It's great to have my websites back up and running, and on Linux!

**Reply by dziudek** [reply]
@tdarais - thank you for spotting this issue - I will add `trim()` before using Site ID in the code to avoid such issues in the future :)
